tests/issues.targets

```diff
@@ -252,6 +252,18 @@
         <ExcludeList Include="$(XunitTestBinBase)/baseservices/varargs/varargsupport_r/*">
             <Issue>Varargs supported on this platform</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/Interop/PInvoke/BestFitMapping/Char/AFT_PFT/AFT_PFT/*">
```

Comment:
Is this disabling these tests completely? Do we have a way to disable tests for JIT stress only?

Reply:
Yes, it disables them completely (for the specific conditions, so Windows x86, Windows x64).

We can disable for ALL platforms for JIT stress by changing their project files to include `<JitOptimizationSensitive>true</JitOptimizationSensitive>`, but that is better used for tests that are really incompatible, not bugs that are hopefully fixed quickly. Plus it means changing more files.

Reply:
I believe we can set <JitOptimizationSensitive>true</JitOptimizationSensitive> for a specific arch/os by adding a condition. It is ugly, but possible.

Reply:
Going forward, I believe it would be better to add. We have the knob, it seems correct to use it.

```
<!-- GH Issue 22436 -->
<JitOptimizationSensitive Condition="'$(OSGroup)' == 'Windows_NT' And '$(BuildArch)' == 'x86'">true</JitOptimizationSensitive>
```